### ISSUE TITLE
update Readme of Installation imgproxy

### ISSUE THREAD

**Issue by r3cha** [forem/forem#12767]
hello, I found that the default URL for imgproxy is `hostname/images` in that case imgproxy Documentation asking to setup
`IMGPROXY_PATH_PREFIX: '/images'` envioement - without this env we get an Invalid signature error.

I found this issue while trying to setup my improxy in Azure k8s cluster - adding this env will fin an Invalid signature error

**Comment by Mr-DG-Wick** [comment]
I'd like to work on this. Can I get some guidance on how to go about this?
At the moment : 
I am reading the docs of imgproxy here : `https://github.com/imgproxy/imgproxy` in order to understand more about the issue.

**Comment by cmgorton** [comment]
Hey @Mr-DG-Wick sorry for the delay in response here. I can assign the issue to you and I will see if I can get some guidance on it for you.

**Comment by Mr-DG-Wick** [comment]
I went to the doc of Imgproxy at `https://developers.forem.com/getting-started/installation/imgproxy`

I guess I just need to the change 
```export IMGPROXY_ENDPOINT='http://localhost:8080'
export IMGPROXY_KEY='1b1c9aae804e070b0864f2547fba7ce8ff31bf7..........'
export IMGPROXY_SALT='8c6d449d4fc2cada5bab538826cae709d2ade9f.........'```

to :
```export IMGPROXY_ENDPOINT='http://localhost:8080'
export IMGPROXY_KEY='1b1c9aae804e070b0864f2547fba7ce8ff31bf7..........'
export IMGPROXY_SALT='8c6d449d4fc2cada5bab538826cae709d2ade9f.........'
export IMGPROXY_PATH_PREFIX: '/images'```

**Comment by Mr-DG-Wick** [comment]
@cmgorton sorry for the ping tho. I assume the team is very busy. But can you please check on this issue when you find time? 
I was just too excited to work on more issues. So, am waiting for this to sort out first :)
thank you.

**Comment by cmgorton** [comment]
Hey @Mr-DG-Wick I have pinged our engineers that are on OSS-rotation this week and someone should get a response to you soon :)

**Comment by Mr-DG-Wick** [comment]
Thanks Christina!!. That would be great help.

**Comment by cmgorton** [comment]
@Mr-DG-Wick I am sorry for any confusion here. 
After discussing this open issue some more with the team we have decided adding this to the docs would not be the best option for us. 

Your suggestion would have been fine for this issue! But upon closer inspection to the original issue we have realized the /image/ proxy we put in place is preconfigured with our forem/infra repo. 

It appears  r3cha  was trying to host Forem and imgproxy on their own and we already have[ selfhost guides](https://github.com/forem/selfhost) to help with this issue. 

Thanks again for your time here and your willingness to contribute to Forem. I am going to close this issue.
